src/odemis/odemisd/modelgen.py

```diff
@@ -842,10 +873,11 @@ def get_instantiables(self, instantiated=None):
                 if psuname not in instantiated:
                     logging.debug("Component %s is not instantiable yet", n)
                     continue
-            for cname in attrs.get("children", {}).values():
-                childat = self.ast[cname]
+            references = attrs.get("children", {}).values() + attrs.get("dependencies", {}).values()
+            for name in references:
```

Comment:
IIRC, here the goal is to check whether all dependency have been instantiated (or, IOW, break if there is any dependency that hasn't been instantiated yet). So the children shouldn't matter (that's why there is the "creator" check).

Reply:
I agree. We still need it for backwards compatibility with the old children though, right?